src/app/components/balance/balance-shimmer.tsx

```diff
@@ -0,0 +1,24 @@
+import { css } from 'leather-styles/css';
+import { Box } from 'leather-styles/jsx';
+
+export function BalanceShimmer({ ...rest }) {
```

Comment:
Needs types

Reply:
Also, this is the skeleton loader? Can we call it skeleton instead? Shimmer makes me think it's just the text effect

Reply:
Also why does this need to be specific to balances? What if we need a skeleton loader for other non-balance text?

Suggest we make a skeleton design system component, with stories, rather than having a specific balance one. This component could wrap that?

Reply:
yes, will rename to Skeleton loader. I actually added already some stories. how you see this `skeleton design system component`? what should it take as props except width/height?